lib/lambda_deployment/configuration.rb

```diff
@@ -22,6 +26,10 @@ def alias_name
       tag
     end
 
+    def environment
+      @environment_cache ||= @config_env.merge(Dotenv.overload(*Dir.glob('.env*')))
```

Comment:
.overload will change the current ENV ... don't you want to just parse the file ?

`Dotenv::Environment.new('.env')`

Reply:
best also have the test double-check that the actual ENV is not changed

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/3d81f2e3-dec8-4395-a284-9c21f770a7e6-948f8120-a27a-42a4-bde2-22298828195d-v1.png?transparent=1&palette=1&width=246)